webpack.static-viz.config.js

```diff
@@ -70,11 +71,16 @@ module.exports = env => {
             },
           ],
         },
+        {
+          test: /\.svg$/i,
```

Comment:
This rule is located below other `SVG` rules that check for certain queries `?source` or `?component` so we don't need to do this like in the main config

https://github.com/metabase/metabase/blob/616c9af6f16e127cd35e19153e1886800571ce27/webpack.config.js#L109

This is needed because the `frontend/src/metabase/plugins/index.ts` imports an `SVG` file which is not used by static viz directly, but static viz still uses that file.

Reply:
it shouldn't affect static viz in any way because this rule is added last, so all the loaders before this newly added one should still take care of everything for static viz.